crates/biome_service/src/workspace/server.rs

```diff
@@ -243,14 +243,22 @@ impl WorkspaceServer {
 
     /// Check whether a file is ignored in the top-level config `files.ignore`/`files.include`
     /// or in the feature `ignore`/`include`
-    fn is_ignored(&self, path: &Path, feature: FeatureName) -> bool {
+    fn is_ignored(&self, path: &Path, features: Vec<FeatureName>) -> bool {
         let file_name = path.file_name().and_then(|s| s.to_str());
+        let ignored_by_features = {
+            let mut ignored = false;
+            for feature in features {
+                // a path is ignored if it's ignored by all features
+                ignored &= self.is_ignored_by_feature_config(path, feature)
```

Comment:
I use a bitwise operation because a path is ignored only if ALL features ignore that path